syntax = "proto3";

package milvus.grpc;

message HelloRequest{
string name = 1;
}

message HelloReply{
string msg = 1;
}

service HelloService{
    rpc SayHello (HelloRequest) returns (HelloReply){}
}